### PR TITLE
Puts medical record quirks in Medical Record "Important Notes", gives each quirk a medical record, cleans up subjects into patients.

### PR DESCRIPTION
**Pull request by RaveRadbury**
## About The Pull Request
Some quirks have med record info for them, others don't. Setting-wise we're already looking at NT being an ethically/morally dubious entity, so I don't see why they wouldn't administer an extensive physical and mental examination to crew before they start working.

I tried to write these from the perspective of a jaded medical professional and took inspiration from things like the Rosenhan experiment to trivialize every little quirk of a spessman into something medically relevant.
 

## Why It's Good For The Game
Generally in-game records don't get updated by players. Giving every quirk a medical record entry should make them more worth taking a look at. 

## Changelog
:cl:
add: Quirks have flavor text in medical records
add: All quirks have medical record entries
spellcheck: All quirk medical records refer to "Patient", removing a few instances of "Subject"
/:cl:

The goal of this PR is to provide each quirk with a medical record, I'm totally open to feedback and suggestions for changing the medical record messages.

### CODE REVIEW

File: code/datums/traits/good.dm [hunk 1]
```diff
@@ -86,6 +93,7 @@
 	mob_trait = TRAIT_MUSICIAN
 	gain_text = "<span class='notice'>You know everything about musical instruments.</span>"
 	lose_text = "<span class='danger'>You forget how musical instruments work.</span>"
+	medical_record_text = "Patient brain scans show a highly-developed auditory pathway"
```
Comment (loserXD) [inline code/datums/traits/good.dm]:
```suggestion
	medical_record_text = "Patient brain scans show a highly-developed auditory pathway."
```

**Review by ghost — CHANGES_REQUESTED** [review]
period

**Comment by Mickyan** [comment]
Well I mean, this'll be nice when someone makes medical_record_text actually do anything..

**Comment by RaveRadbury** [comment]
Ah geeze, guess I gotta take a look at that.

**Comment by RaveRadbury** [comment]
So now the quirk medical records are actually in _Medical Records_. Each quirk record is on it's own newline, it's a little busy (I tried it with maximum quirks) but it's doable for now. Another PR should be made to sort quirks out into major and minor disabilities (maybe even put pineapple preferences under allergies).

**Comment by bgobandit** [comment]
holy shit an actual use for medical records after like a decade

File: code/datums/traits/good.dm [hunk 2]
```diff
@@ -191,6 +206,7 @@
 	gain_text = "<span class='notice'>You feel useless to society.</span>"
 	lose_text = "<span class='danger'>You no longer feel useless to society.</span>"
 	mood_quirk = TRUE
+	medical_record_text = "Patient continues to qualify for welfare and has made no efforts to improve hygeine."
```
Comment (bgobandit) [inline code/datums/traits/good.dm]: hygiene

File: code/datums/traits/negative.dm [hunk 3]
```diff
@@ -233,6 +236,7 @@
 	mob_trait = TRAIT_LIGHT_DRINKER
 	gain_text = "<span class='notice'>Just the thought of drinking alcohol makes your head spin.</span>"
 	lose_text = "<span class='danger'>You're no longer severely affected by alcohol.</span>"
+	medical_record_text = "Patient demonstrates a low tolerance for alcohol.  (Wimp)"
```
Comment (bgobandit) [inline code/datums/traits/negative.dm]: patient adds FUCKING DOUBLE SPACES AFTER SENTENCES AAGGGHHHH

File: code/datums/traits/neutral.dm [hunk 4]
```diff
@@ -38,6 +39,7 @@
 	value = 0
 	gain_text = "<span class='notice'>You feel an intense craving for pineapple.</span>"
 	lose_text = "<span class='notice'>Your feelings towards pineapples seem to return to a lukewarm state.</span>"
+	medical_record_text = "Patient demonstrates an above-average love of pineapple."
```
Comment (bgobandit) [inline code/datums/traits/neutral.dm]: to continue the joke "above-average" should be like "pathological" or something

File: code/datums/traits/good.dm [hunk 5]
```diff
@@ -8,12 +8,14 @@
...
 	gain_text = "<span class='notice'>You feel like you could drink a whole keg!</span>"
 	lose_text = "<span class='danger'>You don't feel as resistant to alcohol anymore. Somehow.</span>"
+	medical_record_text = "Patient demonstrates a high tolerance for alcohol."
 
 /datum/quirk/apathetic
 	name = "Apathetic"
 	desc = "You just don't care as much as other people. That's nice to have in a place like this, I guess."
 	value = 1
 	mood_quirk = TRUE
+	medical_record_text = "Patient ranks high on the Apathy Evaluation Scale."
```
Comment (bgobandit) [inline code/datums/traits/good.dm]: missed joke opportunity, maybe like "Patient was administered the Apathy Evaluation Scale but did not bother to complete it."

**Review by harmonyn — COMMENTED** [review]
some copy edits/missed joke opportunities

**Comment by bgobandit** [comment]
so in general these blurbs are fine but possibly a little on the bland side, I feel like there is opportunity to make them even more horrifying/dystopian, while still having it clear what they refer to

**Comment by RaveRadbury** [comment]
I'm open to further suggestions and if someone wants to do a PR after this rewriting them all I'd be fine with it. Getting it working is my goal here.

**Comment by bgobandit** [comment]
yeah, that came out a little harsher than I intended, I really like the idea and also thank fucking god someone is FINALLY making medical records actually have a use

**Comment by RaveRadbury** [comment]
I'm definitely including your suggestions, I'm just working on another branch right now, else they'd be in already.

**Comment by ExcessiveUseOfCobblestone** [comment]
tfw doesn't use the suggestion feature

**Comment by RaveRadbury** [comment]
@bgobandit I really wanted to do some sort of "Thinks they know better than me/my degree vs your google" joke about obnoxious self-diagnoses for self-aware but I couldn't think of a line that sounded good enough to use.

File: code/datums/traits/neutral.dm [hunk 6]
```diff
@@ -96,6 +100,7 @@
 	gain_text = "<span class='notice'>You feel like you have to stay clean.</span>"
 	lose_text = "<span class='danger'>You no longer feel the need to always be clean.</span>"
 	mood_quirk = TRUE
+	medical_record_text = "While waiting for the exam to begin patient cleaned the examination room."
```
Comment (ExcessiveUseOfCobblestone) [inline code/datums/traits/neutral.dm]:
```suggestion
	medical_record_text = "While waiting for the exam to begin, patient cleaned the examination room."
```

File: code/datums/traits/negative.dm [hunk 7]
```diff
@@ -333,6 +338,7 @@
 	desc = "An accident caused you to lose one of your limbs. Because of this, you now have a random prosthetic!"
 	value = -1
 	var/slot_string = "limb"
+	medical_record_text = "During physical examination patient was found to have a prosthetic limb."
```
Comment (ExcessiveUseOfCobblestone) [inline code/datums/traits/negative.dm]:
```suggestion
	medical_record_text = "During physical examination, patient was found to have a prosthetic limb."
```

File: code/datums/traits/negative.dm [hunk 8]
```diff
@@ -256,6 +260,7 @@
 	name = "Nyctophobia"
 	desc = "As far as you can remember, you've always been afraid of the dark. While in the dark without a light source, you instinctually act careful, and constantly feel a sense of dread."
 	value = -1
+	medical_record_text = "Patient demonstrates fear of the dark. (Seriously?)"
```
Comment (ExcessiveUseOfCobblestone) [inline code/datums/traits/negative.dm]:
```suggestion
	medical_record_text = "Patient demonstrates a fear of the dark. (Seriously?)"
```

File: code/datums/traits/negative.dm [hunk 9]
```diff
@@ -76,7 +77,7 @@
 	value = -1
 	gain_text = "<span class='danger'>You start feeling depressed.</span>"
 	lose_text = "<span class='notice'>You no longer feel depressed.</span>" //if only it were that easy!
-	medical_record_text = "Patient has a severe mood disorder causing them to experience sudden moments of sadness."
+	medical_record_text = "Patient has a severe mood disorder causing them to experience acute episodes of depression."
```
Comment (ExcessiveUseOfCobblestone) [inline code/datums/traits/negative.dm]:
```suggestion
	medical_record_text = "Patient has a severe mood disorder, causing them to experience acute episodes of depression."
```

**Pull request merged**